app/controllers/candidate_interface/course_choices_controller.rb

```diff
@@ -150,6 +150,23 @@ def destroy
       redirect_to candidate_interface_course_choices_index_path
     end
 
+    def add_another_course
+      @additional_courses_allowed = 3 - current_candidate.current_application.application_choices.count
```

Comment:
I don't suppose the magic number 3 is a constant anywhere, is it?

Reply:
There's this https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/master/app/services/candidate_interface/add_course_from_find.rb#L73-L76